Gitcoin's Mission

eswarasai opened the pull request:
<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->

##### Description
<!-- A description on what this PR aims to solve -->
Moved the Gitcoin's Mission and Values to a separate page (`/mission`) which is now linked in both Top Nav as well as Footer Menu.

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, ui, crypto, etc). -->
UI

##### Refers/Fixes
<!--
  Link to an issue if applicable. For example:
  If your PR fixes an issue  -> Fixes: #102
  If your PR refers an issue -> Refs: #101
-->
Fixes: #177

eswarasai commented:
TODO:
1) The top banner in the mockup has `Gitcoin's Mission` but the image in the hero banner is just `Gitcoin`
2) Minor fixes in the responsiveness layout for the Mobile devices

owocki commented:
> The top banner in the mockup has Gitcoin's Mission but the image in the hero banner is just Gitcoin

You could maybe make the positioning of the text relative to the logo configurable.... maybe a boolean template var entitled `position_h1_below_logo` would do the trick..

owocki commented on README.md:
```diff
@@ -296,7 +296,7 @@ DATABASES = {
 virtualenv gcoin
 source gcoin/bin/activate
-pip install -r ../requirements.txt
+pip install -r ../requirements/dev.txt
```
👍 good catch

owocki commented on .gitignore:
```diff
@@ -13,3 +13,4 @@ app/assets/other/avatars/
 venv/
 .coverage
 node_modules/
+gcoin/
```
👍

owocki commented on app/assets/v2/css/gitcoin.css:
```diff
@@ -237,6 +237,14 @@ div.button-pink {
 
+.mission-img-wrapper {
+    height: 250px;
+}
```
some of the images are almost 2500px x 2500px.  mind scaling them down to be less bandwidth hogs?

owocki commented:
left a few comments..  looks good (mostly) to me.  

@pixelantdesign here is a screenshot if u want to give ur own feedback

![screencapture-localhost-8080-mission-1514934552309](https://user-images.githubusercontent.com/513929/34503722-492fe78a-efd7-11e7-864e-8c6de1e8220e.png)

eswarasai commented:
@owocki: I've fixed the mobile responsiveness issue and also updated the images with much lesser resolution now. I've also added the provision of `position_h1_below_logo` in the `minihero.html` but this logo will be `GITCOIN'S` instead of `GITCOIN`.

Let me know if there's anything else pending. Thanks.

PixelantDesign commented:
@eswarasai Looking good! Would it be possible to scale down some of the graphics under “We Value” and “We Interact” so that it’s proportionate to the original mock-up?

eswarasai commented:
@PixelantDesign : The screenshot which you're looking at is not at 100% width of the screen.
@owocki : Correct me if I'm wrong.

So based on the device width, the number of images in a row will be increased/decreased accordingly.

Here's a screenshot on how it looks at full-screen size :

![mission](https://user-images.githubusercontent.com/5172086/34562258-88496296-f173-11e7-9028-846016e4d126.png)

PixelantDesign commented:
@eswarasai that makes sense. Thank you!

One thing I was wondering is where we are linking “Read the mission of Gitcoin” to (since this is the mission page)? If it’s somewhere internal, maybe the button needs renaming? If external, it will open in a new tab?

eswarasai commented:
@PixelantDesign: Yes. Currently the `Read About Our Mission` button is pointed to a [Medium Blog post](https://medium.com/gitcoin/pushing-open-source-forward-for-the-web3-generation-26a5726902ea) which opens up in a new tab.

I think you're right. Since this is the Mission page, the text of the button should be updated to something relevant about the blog post.

@owocki : Any suggestions here?

PixelantDesign commented:
Looking at it again. I think the current language is okay.

It would be great in the future to have clear distinction and set expectations for users whether they are diving into more content on the gitcoin site vs. content that lives outside.
